app/views/layouts/course.html.slim

```diff
@@ -25,4 +25,4 @@
     div.col-lg-10.col-md-9.col-sm-8 class=page_class
       = yield
 
-    #popup-notifier data=controller.next_popup_notification
+    #popup-notifier
```

Comment:
Would it help if we checked for the existence of a `popup_notification` (and/or check if `course_user` is `staff?`), and only render if there is a notification? This way we reduce the number of `fetch` calls every time a user lands on the course homepage.

Reply:
Have added a check for this.